### PR TITLE
board manager: + Agrumino lemon v4

### PR DESCRIPTION
**Pull request by Bighoneypot**
I have add our board esp8266 based. Board name is Lifely Agrumino Lemon v4 (WT8266-S1 Module)

I have edit python script and after  i have running script with option --allgen

I would like to see our board in your json file very soon which I also use with many other esp8266 boards.

### CODE REVIEW

**Comment by Bighoneypot** [comment]
I have this error ESP8266 Arduino CI / Boards.txt check (pull_request) Failing after 21s — Boards.txt check

**Comment by Bighoneypot** [comment]
@d-a-v how long does it take to be inserted in the json for Arduino ??

**Comment by d-a-v** [comment]
In this order:
- 1. this PR has to be merged (in few hours so other maintainers can see it)
- 3. you must wait the official release (that can be a long time - at least 1 - 2 months)
- 2. in the meantime, you can use the "stage" version in PlatformIO or the [unofficial snapshot release](https://d-a-v.github.io/esp8266/Arduino) in the Arduino IDE.

**Review by d-a-v — APPROVED** [review]

**Review by Bighoneypot — COMMENTED** [review]
Bene!

**Comment by Bighoneypot** [comment]
> Bene!

all right!!

File: tools/boards.txt.py
Comment (earlephilhower) [inline tools/boards.txt.py:1465]: This is not needed.  The 2M already has a no-FS option.  Please remove.

File: tools/boards.txt.py
Comment (earlephilhower) [inline tools/boards.txt.py:1496]: Remove.  See line 1444 in the current master.

File: doc/boards.rst
Comment (earlephilhower) [inline doc/boards.rst:223]: Think you've got an unterminated backtick around the "here".

Also some random spacing issues.  Maybe your editr was word-wrapping or something?  Some spaces missing (power line), some extra (this and line 215, for example).

Comment (earlephilhower) [reply doc/boards.rst:223]: Of course, the fix is in the .py and not this generated file, so please fix in boards.txt.py.

File: tools/boards.txt.py
Comment (earlephilhower) [inline tools/boards.txt.py:299]: Is the "WT8266-S1 module" portion really necessary?  This is the longest board name by a big margin...

**Review by earlephilhower — CHANGES_REQUESTED** [review]
Thanks, almost there.

File: tools/boards.txt.py
Comment (d-a-v) [reply tools/boards.txt.py:1465]: This "new" flash-size option allows to hide FS options. Only 2M without FS size appear in the menu.

File: tools/boards.txt.py
Comment (Bighoneypot) [reply tools/boards.txt.py:299]: @earlephilhower Hi, is good only "Lifely Agrumino Lemon v4" ???

**Review by Bighoneypot — COMMENTED** [review]
@earlephilhower I have changed, can you check? Thank you!!

File: tools/boards.txt.py
Comment (Bighoneypot) [reply tools/boards.txt.py]: I have modified, can you check if everything is ok?

File: tools/boards.txt.py
Comment (Bighoneypot) [reply tools/boards.txt.py]: I have modified, can you check if everything is ok?

**Comment by Bighoneypot** [comment]
@d-a-v please, can you help me for this error?

**Comment by d-a-v** [comment]
@Bighoneypot You need to commit boards.txt.py ***and*** boards.txt (the generated changes after `tools/boards.txt.py --allgen`).

**Comment by Bighoneypot** [comment]
> @Bighoneypot You need to commit boards.txt.py _**and**_ boards.txt (the generated changes after `tools/boards.txt.py --allgen`).

@d-a-v  I have commit boards.txt doc/boards.rst and package/package_esp8266com_index.template.json

**Comment by d-a-v** [comment]
@Bighoneypot everything seems fine !

**Comment by Bighoneypot** [comment]
> @Bighoneypot everything seems fine !

@d-a-v now I have to wait for the approval of earlephilhower ?

File: tools/boards.txt.py
Comment (earlephilhower) [inline tools/boards.txt.py:1445]: Never set to anything, should be removed.

File: tools/boards.txt.py
Comment (earlephilhower) [inline tools/boards.txt.py:1491]: Same as before, remove,  please.

**Review by earlephilhower — CHANGES_REQUESTED** [review]
Drop the dead f2m0 refs and we should be good, thanks!

File: tools/boards.txt.py
Comment (Bighoneypot) [reply tools/boards.txt.py]: Line 1445 removed, thank you!!!

File: tools/boards.txt.py
Comment (Bighoneypot) [reply tools/boards.txt.py]: Line 1491 removed, thank you!!!

**Review by earlephilhower — APPROVED** [review]

**Comment by Bighoneypot** [comment]
Thank you @earlephilhower for approving changes!!!!

**Pull request merged**
